src/audio/external.c

```diff
@@ -579,6 +579,8 @@ void func_eu_802e9bec(s32 player, s32 channel, s32 arg2) {
 
 #else
 
+extern s8 sAudioEnabled;
```

Comment:
this is a lil goofy since the `s` implies static, if its not too much of a pain i think it'd be nice if this was changed to `gAudioEnabled` and externed from main.h

Reply:
sAudioEnabled was already nonstatic to begin with. I can rename it though.